cmd/nameset-backfill/main.go

```diff
@@ -71,18 +71,19 @@ func (b *backfiller) run() error {
 
 func (b *backfiller) findCerts() ([]resultHolder, error) {
 	var allResults []resultHolder
+	lastSerial := ""
 	for {
 		var results []resultHolder
 		_, err := b.dbMap.Select(
 			&results,
 			`SELECT c.serial, c.issued, c.expires, c.der FROM certificates AS c
        LEFT JOIN fqdnSets AS ns ON c.serial=ns.serial
        WHERE ns.serial IS NULL
-       ORDER BY c.issued DESC
-       LIMIT ?
-       OFFSET ?`,
+       AND c.serial > ?
+       ORDER BY c.serial
```

Comment:
I know that this has a default direction, but can you include `ASC` anyways? (c.serial HIGHER than the LAST value returned)